### DISCUSSION TITLE
missed out the technical preview access?

### DISCUSSION THREAD

**Discussion by yuvalk-cye** [community/community#11230] [Copilot Conversations] [answered]
A work friend and I signed to the waitlist on the same day, and he got it four months ago, plus told many friends about it lately, and a lot of them got it after a week to two.
I'm six months on the waitlist. I even changed the primary mail couple of times to be sure. And still nothing.
I wondered if the rollout is done, and we will need to wait for the end product?

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
